dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ClaimedTaskRestRepository.java

```diff
@@ -125,6 +135,59 @@ public ClaimedTaskRest findOne(Context context, Integer id) {
         }
     }
 
+    @PreAuthorize("hasAuthority('ADMIN')")
+    @SearchRestMethod(name = "findAllByItem")
+    public Page<ClaimedTaskRest> findAllByItem(@Parameter(value = "uuid", required = true) UUID itemUUID,
+           Pageable pageable) {
+        List<ClaimedTask> tasks = null;
+        try {
+            Context context = obtainContext();
+            Item item = itemService.find(context, itemUUID);
+            if (item == null) {
+                return null;
+            }
+            XmlWorkflowItem xmlWFI = xmlWorkflowItemService.findByItem(context, item);
+            if (xmlWFI == null) {
+                return null;
+            } else {
+                tasks = claimedTaskService.findByWorkflowItem(context, xmlWFI);
+            }
+            return converter.toRestPage(tasks, pageable, utils.obtainProjection());
+        } catch (SQLException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+    }
+
+    @SearchRestMethod(name = "findByItem")
+    public ClaimedTaskRest findByItem(@Parameter(value = "uuid", required = true) UUID itemUUID) {
```

Comment:
This should have a ` @PreAuthorize("hasAuthority('AUTHENTICATED')")` to verify that only logged in users can access this endpoint.  If you add this annotation, you can remove the check below for a `currentUser`, as it's no longer necessary.